src/app/ApplicationTemplate.Shared/CoreStartup.cs

```diff
@@ -75,8 +75,16 @@ private async Task ExecuteInitialNavigation(CancellationToken ct, IServiceProvid
 			{
 				await section.Navigate(ct, () => new OnboardingPageViewModel());
 			}
+#if __MOBILE__ || WINDOWS_UWP
```

Comment:
@vincentcastagna @carlh98 in the template we need to wrap #if and #endif like this (See below in this file). Because the dotnet new code is using #if already for options of the template to know if it needs to add or remove this code.

If we want to keep this code when we run dotnet new, we need the cnd::noEmit
			//-:cnd:noEmit	
#if WINDOWS_UWP || __ANDROID__ || __IOS__	
			//+:cnd:noEmit